Tasks/UseDotNetV2/versioninstaller.ts

```diff
@@ -170,6 +173,14 @@ export class VersionInstaller {
         throw tl.loc("FileNameNotCorrectCompleteFileName", name);
     }
 
+    private _tinyGuid() {
+        return "yxxx".replace(/[xy]/g, function (c) {
+            var r = Math.random() * 16 | 0;
+            var v = c === "x" ? r : (r & 0x3 | 0x8);
```

Comment:
Can you port this to other dotnetcore installer task versions as well

Reply:
Better move to common library.